sqlglot/transforms.py

```diff
@@ -9,6 +9,27 @@
     from sqlglot.generator import Generator
 
 
+def update_from(expression: exp.Expression) -> exp.Expression:
+    """Transform UPDATE FROM to MERGE INTO"""
+    from_expression = expression.find(exp.From)
+    if isinstance(expression, exp.Update) and from_expression:
+        join = from_expression.find(exp.Join)
+        if join:
+            on = join.args.get("on")
+            table = join.find(exp.Table)
```

Comment:
Why do we need `find` here?

```suggestion
            table = join.this
```

Reply:
Let me check this out.